react/src/lib/Popover/index.js

```diff
@@ -190,13 +187,16 @@ class Popover extends React.Component {
       switch (e.which) {
         case 9:
           if (tabbableElements.length) {
-            if (this.props.closeOnFocusLeavesContent) { // if closeOnFocusLeavesContent = true
-              if (e.shiftKey) { // SHIFT + TAB
+            if (this.props.closeOnFocusLeavesContent) {
+              // if closeOnFocusLeavesContent = true
```

Comment:
don't think we need this comment. seems to repeat the code exactly :)

Reply:
i didn't write the comment. formatting the file moved it to the next line :P